app/global/reducers/intervention/sagas/createSession.js

```diff
@@ -17,6 +17,8 @@ export function* createSession({ payload: { id, lastPosition, type } }) {
     type === SessionTypes.SMS_SESSION
       ? 'New SMS Campaign'
       : `${sessionNamePrefix}New Session`;
+  const defaultResponse =
+    type === SessionTypes.SMS_SESSION ? 'Wrong message' : '';
```

Comment:
Why default response is set to 'Wrong message' if Session type is SMS_SESSION?

Reply:
This is the default response send by CIAS if anyone will send a message, which cannot be parsed as an answer. Ideally it should be done on the BE side, but for consistence with other values passed as initial values foe session, I have decided to put it here.